packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.jsx

```diff
@@ -86,9 +86,7 @@ const ContractInfo = ({
     const setHintMessage = () => {
         if (['VANILLALONGCALL', 'VANILLALONGPUT'].includes(type)) {
             return (
-                <Localize
-                    i18n_default_text='The payout at expiry is equal to the payout per point multiplied by the difference between the final price and the strike price.'
-                />
+                <Localize i18n_default_text='The payout at expiry is equal to the payout per point multiplied by the difference between the final price and the strike price.' />
```

Comment:
why do we have changes in trader package?

Reply:
someone force pushed.